go/fix-chain/main/fix-chain.go

```diff
@@ -0,0 +1,116 @@
+package main
```

Comment:
If I'm not mistaken, this will result in the not very inspiring binary name of `main`. :smiley: 

As I recall, you need to put the file that has the `main` package in a directory that bears the name of the binary. See [this](https://github.com/golang/tools/tree/master/cmd/stress), for example (and all their buddies in the `cmd` directory).

And the name of the binary should be something that could make sense in a common `bin` directory, like `ct-fix-chain`, say (`go get` will install it in `$GOPATH/bin`).